frameworks/hdfs/src/main/java/com/mesosphere/sdk/hdfs/scheduler/Main.java

```diff
@@ -37,15 +39,14 @@
 
     public static void main(String[] args) throws Exception {
         if (args.length > 0) {
-            // We manually configure the pods to have additional tasktype placement rules as required for HDFS:
-            new DefaultService(getBuilder(new File(args[0]))).run();
+            SchedulerRunner.fromSchedulerBuilder(getBuilder(new File(args[0]))).run();
         } else {
             LOGGER.error("Missing file argument");
             System.exit(1);
         }
     }
 
-    private static DefaultScheduler.Builder getBuilder(File pathToYamlSpecification)
+    private static SchedulerBuilder getBuilder(File pathToYamlSpecification)
```

Comment:
Yes, there is probably a tradeoff between flexibility and consistency here. Thanks for making things a bit more uniform. This should also help us to identify here things could be automated in the future. (ie.g. a really basic Scheduler shouldn't even need a Main class defined if it's just going to load a `svc.yaml` file). Isn't this what `DefaultExecutor` gives us on the executor side? (my understanding of this may be waaay off though).